.env.example

```diff
@@ -0,0 +1,8 @@
+REACT_APP_VERCEL_GIT_COMMIT_REF=staging # used to track which branch is used to deploy the app. 'staging' uses mainnet for URLs
```

Comment:
This file should already be present in `.env.sample` so we can delete it

Reply:
Deleted!